update terms (#1842)
semgrep/semgrep-docs: src/components/concept/_time-period-filters.md

:::tip 
Quickly view all the findings your organization has fixed in a previous time period, such as a quarter or half-year, by using this filter.
:::

> The time period filter allows you to see which vulnerabilities were opened, fixed, or triaged at a certain period of time. The time period filter is **not** additive; it is a filter operation that precedes other filters in the page. For example, if you select **Last triaged** and select the status **Status Open** filter, no findings appear because by definition, there are no triaged findings that are also open.

Reviewer: ```suggestion
The time period filter allows you to see which vulnerabilities were opened, fixed, or triaged during a certain period of time. The time period filter is **not** additive; it is a filter operation that precedes other filters on the page. For example, if you select **Last triaged** and select the status **Status Open** filter, no findings appear because, by definition, there are no triaged findings that are also open.
```
